de.fu_berlin.inf.dpp.core/test/junit/de/fu_berlin/inf/dpp/git/JGitFacadeTest.java

```diff
@@ -0,0 +1,167 @@
+package de.fu_berlin.inf.dpp.git;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.api.errors.NoFilepatternException;
+import org.eclipse.jgit.errors.AmbiguousObjectException;
+import org.eclipse.jgit.errors.IncorrectObjectTypeException;
+import org.eclipse.jgit.errors.RevisionSyntaxException;
+import org.eclipse.jgit.lib.ObjectId;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class JGitFacadeTest {
```

Comment:
Did you verified whether your tests could be simplified by the [jgit junit tools](https://download.eclipse.org/jgit/site/5.1.3.201810200350-r/apidocs/org/eclipse/jgit/junit/package-summary.html)?

Reply:
I see no need